backend/src/test/java/de/neuefische/readandmeet/backend/service/MeetingServiceTest.java

```diff
@@ -63,4 +64,17 @@ void expectMeeting_whenPOSTingMeeting() {
         verify(uuIdService).getRandomId();
         verify(meetingRepo).insert(expected);
     }
+    @Test
+    void expectMeeting_whenMeetingById() {
+        //GIVEN
+        Meeting expected = new Meeting("123", "book", LocalDate.now(), "online");
+
+        //WHEN
+        when(meetingRepo.findById("abc")).thenReturn(Optional.of(expected));
+        Meeting actual = meetingService.getDetails("abc");
+
+        //THEN
+        assertEquals(expected, actual);
+        verify(meetingRepo).findById("abc");
```

Comment:
```suggestion
        verify(meetingRepo).findById("abc");
        assertEquals(expected, actual);
```

You can consider putting the verify statement before the assertEquals statement. In case the test fails, you can then at least see if the the repo mock was performed correctly.

In your case, if the assert fails, you can not see if the repo call was correct, because the test stops.

All in all it's a matter of personal preference :)

Reply:
thank you, I will definitely consider that!